src/web/mjs/connectors/MaID.mjs

```diff
@@ -16,8 +10,20 @@ export default class MaID extends WordPressEManga {
         this.url = 'https://www.maid.my.id';
         this.path = '/manga-list/?list';
 
-        this.queryMangas = 'div.contentpost div.soralist ul li a.series';
-        this.queryChapters = 'div.bxcl ul li span.lchx a';
-        this.queryPages = 'div#readerarea :not(.kln) > source';
+        this.queryMangas = 'div.mangalist-blc ul li.Manga a.series';
+        this.queryChapters = 'div.series-chapter ul.series-chapterlist li div.flexch-infoz a';
+        this.queryPages = 'div.reader-area source[src]:not([src=""])';
+    }
+
+    async _getChapters(manga) {
+        let request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryChapters);
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, this.url),
+                title: element.title.replace(manga.title, '').trim(),
+                language: ''
```

Comment:
May remove unused var.

Reply:
Will be kept for semantically correctness